DEVELOPER.MD

```diff
@@ -47,47 +46,49 @@ There are at least 7 containers that will be running:
 | Airflow Data Collection Worker  | patentsview:airflow | Data Collection Server | docker-compose-data-collection-worker.yaml |    pv_data_pipeline_data_collection_worker |     8793/8793                                   | airflow celery worker -q data_collector |
 | Airflow Disambiguation Worker  | patentsview:airflow | Disambiguation Server  | docker-compose-disambiguator.yaml                                          |    pv_data_pipeline_disambiguation_worker |                          8793/8793              | airflow celery worker -q disambiguation |
 
-### airflow_pipeline_env.sh
-
-- Sensitive variables should be stored in `airflow_pipeline_env.sh`. See `airflow_pipeline_env.sh.template` for
-  reference
-- The variables need to be same in each server where any pipeline container is deployed
-
-## init-sample.sql
+### init-sample.sql
 
 - Make a copy of this file as airflow-metadata-db-sql/init.sql
 - Replace placeholder password with a generated password. You'll use this in the next section
 
-## docker-compose-db.yaml
+### docker-compose-db.yaml
 
 - Contains the configuration required to start the database. This includes
     - An init sql that creates the database and username required for airflow (Use password from previous section)
     - The root password configuration
     - Mounted volume configuration where the database files are stored
 - See `docker-compose-db.yml.template` for reference
 
-## docker volume (Developer machine)
+### docker volume (Developer machine)
 
 - On developer machines (non-production), you'll need to create the data volume like so:
   `docker volume create pv_update_volume`
+- Volumes connect to the host server so that data will be saved. Otherwise, everything in the container is deleted everytime the container is closed.
+
+| Volume      | Purpose                                                 | Servers                | 
+|-------------|---------------------------------------------------------|------------------------|
+| data-volume | Store XML rawdata                                       | PV-DataCollector; PVGP |
+| app-volume  | Store PGPUBS XML rawdata                                    | PV-DataCollector; PVGP |
+| archive     | Store archived weekly PV databases (e.g. upload_20100422) | PV-DataCollector; PVGP |
 
-## docker network create
+### docker network create
 
 - The first time you set this up on each machine, you'll need to create a network using the command
   `docker network create patentsview-pipeline-network`
 
-## .env.template (Only on Linux machines)
+### .env.template (Only on Linux machines)
 
 - Make a copy of .env.template to .env and se the `HOST_UID` value to the result of running this command: `id -u`
+- Note that the airflow container MUST HAVE THE SAME UID as the server and that the GID MUST BE SET TO 0
```

Comment:
Good note! Recommend adding documentation reference: https://airflow.apache.org/docs/docker-stack/entrypoint.html#allowing-arbitrary-user-to-run-the-container